blocks/amp-story/text/edit.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { PanelBody, SelectControl, withFallbackStyles } from '@wordpress/components';
+import { RichText, InspectorControls, FontSizePicker, withFontSizes } from '@wordpress/editor';
+import { compose } from '@wordpress/compose';
+
+const { getComputedStyle } = window;
+
+/**
+ * Internal dependencies
+ */
+import getTagName from './getTagName';
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { fontSize, customFontSize } = ownProps.attributes;
+	const editableNode = node.querySelector( '[contenteditable="true"]' );
```

Comment:
Querying for `[contenteditable="true"]` seems a bit brittle. Is the node not accessible in another way?